versioned_docs/version-21.10/releases/centreon-core.md

```diff
@@ -141,17 +141,45 @@ By:
 - Move to PHP 8.0
 - Preparing Debian 11 support
 
-## Centreon Engine
+##  Centreon Collect
+
+> As of the 21.10.0 version, the components of Centreon Collect (Centreon Broker, Centreon Clib, Centreon Engine and Centreon Connectors)
+> are released concurrently.  They are now grouped under this section.
```

Comment:
```suggestion
> are released together. They are now grouped under this section.
```

Reply:
this may lead to confusion where users could think that they are released all in one centreon-collect package instead of the different components' packages

Reply:
oh, I thought they were.